e2e/fixtures/issueResolution.json

```diff
@@ -319,11 +319,7 @@
       {
         "id": "749f29dc-c200-4377-9e12-a5b4bad631b3",
         "value": {
-          "order":{
-            "document_url":"http://dm-store:8080/documents/7196dcc4-dd17-49ba-9924-7480b91e620e",
-            "document_filename":"mockFile.pdf",
-            "document_binary_url":"http://dm-store:8080/documents/7196dcc4-dd17-49ba-9924-7480b91e620e/binary"
-          },
+          "order":"TO BE FILLED WITH VALID DATA",
```

Comment:
do we need it at all? 
it is overriden anyway

Reply:
Was just being consistent with everything else